posts/2022-12-06-automation-star-conference.adoc

```diff
@@ -0,0 +1,132 @@
+++++
+</br>
+++++
+Conference was focused mainly on test automation which covered a variety of different topics such as event driven test automation, cypress, test automation frameworks, strategy, functional testing, codeless test automation and soft skills of automation.
+
+== Things before the Conference
+
+One week before the conference an email was sent to all the attendees which has the instructions about the conference. The email also contained the link for the https://whova.com/portal/autom1_202210/?source=download_page/[whova app] where attendees can get live updates and chat with others. After registering on the app, all the information about the sessions and the speakers were available. Attendees can also initiate the discussion and can contact others like the ice breaking sessions. One interesting thing about the app was the Photo competition, where participants can post the conference pictures and picture with the maximum likes will receive the price.
+
+== Conference Program
+
+The https://automation.eurostarsoftwaretesting.com/conference/2022/programme/[conference program] included two days of Tutorials, keynotes, and talks led by experts from around the testing world. It also included exclusive networking and social events like lean coffee and Community Conversations. Prior to the conference, attendees have the option to choose the Tutorials and the sessions in the Whova App and from the app they can also download the slides for the presentations. The tutorials and talks were in German and English so attendees can select according to their preferred language. The tutorials were scheduled for half day and the talks were for forty-five minutes.
+
+== Begins the Conference
+
+The first day of the conference started with the Badge and goodies collection and morning coffee. Then attendees were directed to their selected tutorials which were scheduled for half day. Second half of the first day and the second day of the conference was the combination of Keynotes and talks. All attendees get to choose between different talks happening at the same time in separate conference rooms. At a time participants can attend one out of five talks.
+
+== Talks at Conference:
+
+In the following, I would like to briefly discuss the talks that we have attended:
+
+== 1. Making Great Waves: 8 Software Testing Convictions
+
+Testing is an art as much as Engineering, we can take cues from prolific artists in how we approach quality in our own work. In this keynote *Andrew Knight* mentioned how we can create our own great waves in software testing. He covered following eight auspicious ideas which were inspired by ukiyo-e printing:
+
+- Focus on behavior
```

Comment:
In ascii doctor, you should use `*` for unordered list: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#lists